gulpfile.js

```diff
@@ -147,7 +147,7 @@ gulp.task("critical-front", function(cb) {
     coreGulpConfig = require(util.env.epiq_dir + "/gulpconfig.json"),
     gulpconfig = require(util.env.gulpconfig),
     urls = {
-      site: util.env.site,
+      site: gulpconfig.site,
```

Comment:
Just an idea:

Could we pull this info out this file/repo? https://github.com/jobiqo/ce_scripts/blob/master/inventory/sites.yml

Reply:
Nope not really. We can't, we discussed this morning in the chat. Because we have sites that have multiple urls and themes..and each theme has its own url (e.g. pharma is not in that list..)